components/datadog/agent/kubernetes_helm.go

```diff
@@ -292,36 +292,36 @@ func buildLinuxHelmValues(installName, agentImagePath, agentImageTag, clusterAge
 				"agent": pulumi.Map{
 					"resources": pulumi.StringMapMap{
 						"requests": pulumi.StringMap{
-							"cpu":    pulumi.String("300m"),
+							"cpu":    pulumi.String("600m"),
 							"memory": pulumi.String("500Mi"),
```

Comment:
@vboulineau : A memory request of 500 MiB looks generous.
But the real measured consumption isn’t a lot less than that: 
![image](https://github.com/DataDog/test-infra-definitions/assets/1437785/e5a39da6-8d44-49b5-b106-31616902a66a)
Also, on the Kind node, with the `t3.xlarge` type, our usage of memory is lower that our usage in CPU:
```
kubectl describe node/lenaic-kind-control-plane
[…]
  Resource           Requests      Limits
  --------           --------      ------
  cpu                2156m (53%)   3880m (97%)
  memory             3090Mi (19%)  3802Mi (24%)
```

Reply:
What about the 0.6 CPU usage? Seems high for `kind` but perhaps SBOM scanning activated?

Reply:
For the CPU, here is the past observed consumption:
![image](https://github.com/DataDog/test-infra-definitions/assets/1437785/54d214b3-e6b9-4c32-b7c6-7cd5955658ff)
![image](https://github.com/DataDog/test-infra-definitions/assets/1437785/34867084-458c-4e53-a163-1602cac206d3)
I’m not sure to understand why we see those outliers above 400 millicores.
SBOM is indeed activate, but so it is for every single test run.
Given the fact that CPU is elastic, we can make a try with a request at 400 millicores ?